core/commonMain/src/kotlinx/io/Output.kt

```diff
@@ -14,7 +14,10 @@ import kotlinx.io.pool.*
 public abstract class Output(
     protected val pool: ObjectPool<Buffer> = DefaultBufferPool.Instance
 ) : Closeable {
-    internal val bufferPool: ObjectPool<Buffer> get() = pool
+    /**
+     * Buffer pool used by this [Input] for copy and cache operations.
+     */
+    val bufferPool: ObjectPool<Buffer> get() = pool
```

Comment:
1) Why do we need it `public`?
2) explicit `public`
3) What's the difference between `pool` and `bufferPool`? If we have to make it public, why don't we stick with a single getter?

Reply:
1. We need it `public` to use in transformers.
3. I'll keep the one from the constructor.